server/xpub-server/entities/manuscript/resolvers/uploadManuscript.js

```diff
@@ -1,181 +1,19 @@
 const config = require('config')
+const { User } = require('@elifesciences/xpub-model')
+const { Manuscript, S3Storage } = require('@elifesciences/xpub-controller')
 const pubsubManager = require('pubsweet-server/src/graphql/pubsub')
-const logger = require('@pubsweet/logger')
-const scienceBeamApi = require('../../../../xpub-controller/client/scienceBeamApi')
-
-const { ON_UPLOAD_PROGRESS } = pubsubManager.asyncIterators
-
-const { Manuscript, User } = require('@elifesciences/xpub-model')
-const { File } = require('@elifesciences/xpub-model')
-const { S3Storage } = require('@elifesciences/xpub-controller')
+const ScienceBeamApi = require('../../../../xpub-controller/client/scienceBeamApi')
```

Comment:
This path should be `@elifesciences/xpub-controller`, but the API hasn't been exported yet. We should do this in a separate PR.

Reply:
Addressed by https://github.com/elifesciences/elife-xpub/pull/1485